Singularity/Singularity/Map/Map.cs

```diff
@@ -227,6 +230,26 @@ public static bool IsOnTop(Rectangle rect, Camera camera = null)
 
         }
 
+        public void KeyTyped(KeyEvent keyEvent)
+        {
+            foreach (var key in keyEvent.CurrentKeys)
+            {
+                if (key == Keys.F4)
+                {
+                    mDebug = !mDebug;
+                }
+            }
+        }
+
+        public void KeyPressed(KeyEvent keyEvent)
+        {
+            
+        }
+
+        public void KeyReleased(KeyEvent keyEvent)
```

Comment:
Adding a TODO: DEBUG-remove here for example

Reply:
true, i agree on that. Would definitely make sense